types/foundry-vtt-api/types/ui/dragdrop.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A controller class for managing drag and drop workflows within an Application instance.
+ * The controller manages the following actions: dragstart, dragover, drop
+ * @see {@link Application}
+ *
+ * @param dragSelector	The CSS selector used to target draggable elements.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/ui/dragdrop.d.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.